pkg/contracts/annotation.go

```diff
@@ -52,6 +52,12 @@ func getTagValue(layer LayerType) string {
 	switch layer {
 	case Application:
 		return os.Getenv(TagEnvKey)
+	case Os:
+		hostname, _ := os.Hostname()
+		return hostname
+	case Host:
+		hostname, _ := os.Hostname()
```

Comment:
@michaelehab The hostname is not guaranteed to be unique, especially in the case of a VM farm. Instead, for a non-Dell Linux environment, this should be the contents of the `/etc/machine-id` file.